action.yml

```diff
@@ -1,5 +1,5 @@
-name: '@guardian/post-release-action'
-description: 'A GitHub action to handle the post release process for repositories with branch protection'
+name: '@guardian/actions-merge-workflow-changes-to-protected-branch'
+description: 'A GitHub action to open, approve and merge a PR with changes generated during a workflow action on a protected branch'
```

Comment:
```suggestion
description: 'A GitHub action to open, approve and merge a PR of changes generated during a workflow action on a protected branch'
```

Reply:
I've done this as a separate commit to change it in both the `action.yml` file and the `package.json`